framework/src/utils/PetscSupport.C

```diff
@@ -968,6 +968,26 @@ colorAdjacencyMatrix(PetscScalar * adjacency_matrix,
   ISColoringDestroy(&iscoloring);
 }
 
+bool
+detectHypreFromOptions()
+{
+  char pc_type[256];
+  PetscBool flg;
```

Comment:
It's a dang good thing you didn't type that "a". You saved soooo much space there!

Reply:
![image](https://user-images.githubusercontent.com/202302/60052733-472cfb80-9693-11e9-92b2-2012ba73a6db.png)